### ISSUE TITLE
Results contain hex notation

### ISSUE THREAD

**Issue by rguikers** [eris-ltd/eris-contracts.js#45]
In version 0.15.3 all numeric results (uintXX and addresses) are returned as hex code (preceding with 0x).
Expected a numeric type, not a hex string.

**Comment by NodeGuy** [comment]
I'm not seeing this in the [HTTP.js integration test](https://github.com/eris-ltd/eris-contracts.js/blob/87503627f6f4d5b9bc8cba997e6094d474882bb4/test/integration/HTTP.js#L35): `value` is coming back as `42`, not a string.

Would you please give me some sample code?

**Comment by nottherealbatman** [comment]
@NodeGuy 
See testcase:

Contract:
    contract Test {

	    string _name;

	    function add(int a, int b) constant returns (int sum) {
	  	    sum = a + b;
	    }

	    function setName(string newname) {
		   _name = newname;
	    }

	    function getName() returns (string) {
		    return _name;
	    }
    }


JavaScript:

		var myContractFactory = contractManager.newContractFactory(abi);
		myContractFactory.new({data: dat}, function(error, contract){
		    if (error) {
		        // Something.
		        throw error;
		    }
		    
			contract.setName('Batman', function(err, batman){
				console.log('batman', batman);
                      // returns batman,  0x0

				contract.getName(function(err, name){
					console.log('name', name)
                      // returns 0x000000000000000000000000000000000000000000000000000000000000002000000000000000000000000000000000000000000000000000000000000000064261746D616E0000000000000000000000000000000000000000000000000000
				})
			})

		    		
		});

**Comment by NodeGuy** [comment]
Fixed in 0.15.4.

**Comment by rguikers** [comment]
For me the following contract :
```
contract Test {

    function getAddress() returns (address) {
      return this;
    }

    function getNumber() returns (uint) {
      return 100;
    }

}
```

with code : 

```
contractsManager = erisC.newContractManagerDev(erisDbUrl, account);
contractAPI = contractsManager.newContractFactory(testAbi).at(ppuContractAddress);

contractAPI.getAddress(function(err, data) {
  if(err){
    console.log("ERROR");
    console.log(err);
  } else {
    console.log("getAddress : " + data);
  }
})
contractAPI.getNumber(function(err, data) {
  if(err){
    console.log(err);
  } else {
    console.log("getNumber : " + data);
  }
})
```

returns : 
getAddress : undefined
getNumber : undefined
in version 0.15.4

returns : 
getNumber : 100
getAddress : 1E6F1044C0AFBF317F15C40177C83172E4A8B3D6
in version 0.13.3

Both tested on 0.12RC2, will start the same test in 0.12 asap.

**Comment by NodeGuy** [comment]
Fixed in 0.15.5.
